### ISSUE TITLE
Equation Explorer shows Equation4270 wrong

### ISSUE THREAD

**Issue by lyphyser** [teorth/equational_theories#480]
It says: "Equation4270[x ◇ (x ◇ x) = x ◇ (y ◇ z)]"

But the equation actually is:
equation 4270 := x ◇ (x ◇ x) = x ◇ (y ◇ y)

The final y has been incorrectly replaced with a z.

**Comment by carlini** [comment]
It looks like all may be broken. I will take a look to see what's going on soon.

**Comment by carlini** [comment]
equation 953 has been deleted from deleted from AllEquations:

https://github.com/teorth/equational_theories/blob/b0708a3031b250ad0f228b3a4b57530031b274ae/equational_theories/AllEquations.lean#L972-L975

I'll bring it back from the dead.

**Comment by carlini** [comment]
claim

**Comment by carlini** [comment]
propose PR #481

**Comment by goens** [comment]
equation 953 didn't disappear, I moved it to `Equations.lean` in https://github.com/teorth/equational_theories/pull/460

It's still right there where it should: https://github.com/teorth/equational_theories/blob/b0708a3031b250ad0f228b3a4b57530031b274ae/equational_theories/Equations.lean#L83

It shouldn't be moved again, because it's in Chapter 2 of the blueprint (see also the corresponding Zulip thread https://leanprover.zulipchat.com/#narrow/stream/458659-Equational/topic/Equations.2Elean.20vs.20Chapter.202)

Maybe the script that breaks is not using `Equations.lean`?  (The lean AllEquations file does import it, but if an external script ignores this it might still break)

Also note that this conflicts with #472 and would probably be fixed by that anyway

**Comment by goens** [comment]
I did just notice that #481 only adds it back as a comment. I guess that works (it's a bit hacky), but we should probably coordinate it with #472 anyway (it's going to conflict)

**Comment by carlini** [comment]
Yeah, but every other equation that has been removed from AllEquations.lean was removed in this way. I agree it's hacky but it's how things have been done in all the prior cases. Long term plan is to load from a .json file for all scripts that need equation data, that should be done this week I think.

I closed my prior PR because it was put back in the split of AllEquations and have now opened a new PR that'll raise an error if this happens again.
